templates/solutions/automotive/buyers-guide/index.html

```diff
@@ -0,0 +1,785 @@
+    <div class="u-fixed-width u-no-padding">
+      {{ image (
+      url="https://assets.ubuntu.com/v1/e8c27f33-GettyImages-1186091802.jpg",
+      alt="",
+      width="2121",
+      height="906",
+      hi_def=True,
+      loading="auto",
+      attrs={"class": "banner-image--non-interactive"},
+      ) | safe
+      }}
+    </div>
+  </section>
+
+  <section class="p-strip u-no-padding--top is-shallow u-hide-on-reveal">
+    <div class="u-fixed-width">
+      <hr class="p-rule is-dark" />
+      <h3 class="p-muted-heading">Criteria for choosing an automotive-grade Linux</h3>
+    </div>
+  </section>
+
+  <section class="feature p-strip u-no-padding--top" id="compatibility">
+    <div class="u-fixed-width p-card">
+      <div class="u-show-on-reveal">
+        <div class="row">
+          <div class="col-9 col-medium-4 col-small-3">
+            <p>Automotive OS buyer's guide</p>
+          </div>
+          <div class="col-3 col-medium-2 col-small-1 u-align--right">
+            <p>
+              <a href="#" class="p-icon--close">Close</a>
+            </p>
+          </div>
+          <hr class="p-rule is-dark" />
+        </div>
+        <div class="row">
+          <div class="col-6 col-medium-3 col-small-2">
+            <p class="p-muted-heading">Previous</p>
+            <p class="p-heading--4">
+              <a href="#">&lsaquo;&nbsp;Overview</a>
+            </p>
+          </div>
+          <div class="col-6 col-medium-3 col-small-2 u-align--right">
+            <p class="p-muted-heading">Next</p>
+            <p class="p-heading--4">
+              <a href="#security">Security&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <hr class="p-rule is-dark" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-6">
+          <h2>Compatibility</h2>
+        </div>
+        <div class="col-6">
+          <p class="u-limit-content">
+            Considering the number of different chipsets present in a vehicle today, it is important that the Linux distribution you choose is compatible with a maximum number of hardware platforms. We make sure to provide best level support and integration to hardware components that are widely used in the automotive industry. Thanks to our alliances and enablement teams, we work closely with silicon providers to make sure that Ubuntu runs seamlessly on top of their hardware.
+          </p>
+          <p class="u-hide-on-reveal">
+            <a class="js-section-select" href="#compatibility">Continue reading about compatibility&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <div class="row p-block u-sv3">
+        {{ image (
+        url="https://assets.ubuntu.com/v1/afce18b9-rfq-_0005_AdobeStock_621962476.jpg",
+        alt="",
+        width="2496",
+        height="1066",
+        hi_def=True,
+        loading="lazy",
+        attrs={"class": "banner-image u-sv3 u-show-on-reveal"},
+        ) | safe
+        }}
```

Comment:
```suggestion
        {{ image(url="https://assets.ubuntu.com/v1/afce18b9-rfq-_0005_AdobeStock_621962476.jpg",
                        alt="",
                        width="2496",
                        height="1066",
                        hi_def=True,
                        loading="lazy",
                        attrs={"class": "banner-image u-sv3 u-show-on-reveal"}) | safe
        }}
```
Can you please update all the image tags to follow the standard formatting?